src/UartSerial.cpp

```diff
@@ -156,7 +157,7 @@ UartSerial::begin (
     // Leave all output flags unset
 
     // c_cflag - control modes
-    _tio_config.c_cflag |= CS8;  //TODO: Configuration passed by caller
+    _tio_config.c_cflag |= c_cflags;
```

Comment:
On line 72, we set EVERY flag, then re-set the flags in the break statement, then OR them into the final result, which will always result all flags set, regardless of the options.

I think we should initialize `size_t c_cflags = 0;`, then after you've selected the desired flags in the `switch`, then set `_tio_config.c_cflag = c_cflags;` instead of `_tio_config.c_cflag |= c_cflags;`